addons/calendar/static/src/views/fields/many2many_attendee_expandable.js

```diff
@@ -19,6 +19,9 @@ export class Many2ManyAttendeeExpandable extends Many2ManyAttendee {
         this.uncertainCount = this.attendeesCount - this.acceptedCount - this.declinedCount;
 
         useEffect(() => {
+            if (this.env.isSmall) {
+                return;
+            }
```

Comment:
The traceback will still occur if you

1) open the view in mobile
2) click on an event
3) resize the window so that it's no longer mobile
4) click on the Many2ManyAttendeeExpandable arrow

Until we support dynamically resizing the window, you can avoid the traceback by performing the check only once:

```suggestion
        if (!this.env.isSmall) useEffect(() => {
```

Additionally, this would be easier to read and debug.

Reply:
You're right this way is more readable, thanks for the review! 😁